plugins/dell-dock/meson.build

```diff
@@ -14,7 +14,6 @@ shared_module('fu_plugin_dell_dock',
     'fu-dell-dock-hub.c',
     'fu-dell-dock-i2c-tbt.c',
     'fu-dell-dock-i2c-mst.c',
```

Comment:
is it ok to leave a comma here ?

Reply:
Yeah this is generally done so that you can add a new file after this later without needing to change this line to add another comma.